<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Medication_00</class>
 <widget class="QWizardPage" name="Medication_00">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>447</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="medicationLabel">
     <property name="toolTip">
      <string comment="MEDICAITON EDIT TOOLTIP">MEDICATION</string>
     </property>
     <property name="text">
      <string comment="MEDICATION EDIT LABEL">MEDICATION</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="medicationField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="MEDICATION EDIT TOOLTIP">MEDICATION</string>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="genericLabel">
     <property name="toolTip">
      <string comment="GENERIC EDIT TOOLTIP">GENERIC</string>
     </property>
     <property name="text">
      <string comment="GENERIC EDIT LABEL">GENERIC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="genericField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="GENERIC EDIT TOOLTIP">GENERIC</string>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="manufacturerLabel">
     <property name="toolTip">
      <string comment="MANUFACTURER EDIT TOOLTIP">MANUFACTURER</string>
     </property>
     <property name="text">
      <string comment="MANUFACTURER EDIT LABEL">MANUFACTURER</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="manufacturerField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="MANUFACTURER EDIT TOOLTIP">MANUFACTURER</string>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ndcLabel">
     <property name="toolTip">
      <string comment="NDC EDIT TOOLTIP">NDC</string>
     </property>
     <property name="text">
      <string comment="NDC EDIT LABEL">NDC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="ndcField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="NDC EDIT TOOLTIP">NDC</string>
       </property>
       <property name="maxLength">
        <number>14</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="strengthLabel">
     <property name="toolTip">
      <string comment="STRENGTH EDIT TOOLTIP">STRENGTH</string>
     </property>
     <property name="text">
      <string comment="STRENGTH EDIT LABEL">STRENGTH</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="strengthField">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="STRENGTH EDIT TOOLTIP">STRENGTH</string>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="formLabel">
     <property name="toolTip">
      <string comment="FORM EDIT TOOLTIP">FORM</string>
     </property>
     <property name="text">
      <string comment="FORM EDIT LABEL">FORM</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QComboBox" name="formField">
       <property name="toolTip">
        <string comment="FORM EDIT TOOLTIP">FORM</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="dispenseLabel">
     <property name="toolTip">
      <string comment="DISPENSE UNITS EDIT TOOLTIP">DISPENSE UNITS</string>
     </property>
     <property name="text">
      <string comment="DISPENSE UNITS EDIT LABEL">DISPENSE UNITS</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="dispenseUnitsField">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="DISPENSE UNITS EDIT TOOLTIP">DISPENSE UNITS</string>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="unitSizeLabel">
     <property name="toolTip">
      <string comment="UNIT SIZE EDIT TOOLTIP">UNIT SIZE</string>
     </property>
     <property name="text">
      <string comment="UNIT SIZE EDIT LABEL">UNIT SIZE</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLineEdit" name="unitSizeField">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="UNIT SIZE EDIT TOOLTIP">UNIT SIZE</string>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>medicationField</tabstop>
  <tabstop>genericField</tabstop>
  <tabstop>manufacturerField</tabstop>
  <tabstop>ndcField</tabstop>
  <tabstop>strengthField</tabstop>
  <tabstop>formField</tabstop>
  <tabstop>dispenseUnitsField</tabstop>
  <tabstop>unitSizeField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
